service/app/handlers_app_user.go

```diff
@@ -35,11 +35,14 @@ import (
 	"github.com/CanonicalLtd/serial-vault/crypt"
 	"github.com/CanonicalLtd/serial-vault/datastore"
 	"github.com/CanonicalLtd/serial-vault/random"
+	svlog "github.com/CanonicalLtd/serial-vault/service/log"
```

Comment:
is this prefix needed?. I don't see any other log package here.

Reply:
Doesn't look like it. I must have replaced all the original log.Printf commands.

Reply:
Actually it is used, but hidden in this diff (line 25)

Reply:
ok, thanks